src/domain/common/utils/__tests__/safe.spec.ts

```diff
@@ -46,9 +86,264 @@ function safeTxHashMultisigTransactionBuilder(): IBuilder<BaseMultisigTransactio
 }
 
 describe('Safe', () => {
-  it.todo('getBaseMultisigTransaction');
+  describe('getBaseMultisigTransaction', () => {
+    it('should return a valid BaseMultisigTransaction', () => {
+      const multisigTransaction = multisigTransactionBuilder().build();
+
+      const result = getBaseMultisigTransaction(multisigTransaction);
+
+      expect(result).toEqual({
+        to: multisigTransaction.to,
+        value: multisigTransaction.value,
+        data: multisigTransaction.data,
+        operation: multisigTransaction.operation,
+        safeTxGas: multisigTransaction.safeTxGas,
+        baseGas: multisigTransaction.baseGas,
+        gasPrice: multisigTransaction.gasPrice,
+        gasToken: multisigTransaction.gasToken,
+        refundReceiver: multisigTransaction.refundReceiver,
+        nonce: multisigTransaction.nonce,
+      });
+    });
+  });
+
+  describe('getSafeMessageMessageHash', () => {
+    describe('generates a valid messageHash', () => {
+      it('should handle strings', () => {
+        const chainId = faker.string.numeric();
+        const safe = safeBuilder().build();
+        const message = faker.lorem.sentence();
+
+        expect(() =>
+          getSafeMessageMessageHash({
+            chainId,
+            message,
+            safe,
+          }),
+        ).not.toThrow();
+      });
+
+      it('should handle typedData', () => {
+        const chainId = faker.string.numeric();
+        const safe = safeBuilder().build();
+        const message = buildTypedData().build();
 
-  it.todo('getSafeMessageMessageHash');
+        expect(() =>
+          getSafeMessageMessageHash({
+            chainId,
+            message,
+            safe,
+          }),
+        ).not.toThrow();
+      });
+
+      it.each(DOMAIN_WITHOUT_CHAIN_ID_VERSIONS)(
+        "should handle versions that don't include the chainId in the domain (%s)",
+        (version) => {
+          const chainId = faker.string.numeric();
+          const safe = safeBuilder().with('version', version).build();
+          const message = faker.helpers.arrayElement([
+            faker.lorem.sentence(),
+            buildTypedData().build(),
+          ]);
+
+          expect(() =>
+            getSafeMessageMessageHash({ chainId, message, safe }),
+          ).not.toThrow();
+        },
+      );
+
+      it.each(DOMAIN_WITH_CHAIN_ID_VERSIONS)(
+        'should handle versions that include the chainId in the domain (%s)',
+        (version) => {
+          const chainId = faker.string.numeric();
+          const safe = safeBuilder().with('version', version).build();
+          const message = faker.helpers.arrayElement([
+            faker.lorem.sentence(),
+            buildTypedData().build(),
+          ]);
+
+          expect(() =>
+            getSafeMessageMessageHash({ chainId, message, safe }),
+          ).not.toThrow();
+        },
+      );
+    });
+
+    describe.skip('examples', () => {
```

Comment:
As mentioned in the desc., these will be added later (as well as those for `safeTxHash`es) because they take some time to find.